schema.json

```diff
@@ -1214,7 +1148,8 @@
                 }
               }
             }
-          }
+          },
+          "additionalProperties": false
```

Comment:
@stiebitzhofer  In all the test related properties inside the `Inspection`, i added `additionalProperties = false` since i saw examples that used properties like C13.1, that cannot be rendered properly in the template. 
Do you agree with that change ?

Reply:
Yes (Remains from the past :-()